XPlat/ExposureNotification/source/Xamarin.ExposureNotification/ExposureNotification.ios.cs

```diff
@@ -275,19 +354,73 @@ static async Task<(ExposureDetectionSummary, Func<Task<IEnumerable<ExposureInfo>
 				}
 			}
 
+			// Branch the logic to either return the new data, or fall back to the old info
+			if (IsDailySummaries)
+			{
+				// Check that the summary has actual data before notifying the callback
+				if (detectionSummary.DaySummaries.Length > 0)
+				{
+					var (windows, dailySummary) = await PlatformProcessSummaryV2Async(m, detectionSummary, cancellationToken);
```

Comment:
Hello, I am actively working on testing the Nuget package with these changes. Based on what I see on iOS,  	`await DailySummaryHandler!.ExposureStateUpdatedAsync(windows, dailySummary)` is never called, indicating that something here might be out of control. `ExposureDetectedAsync` is not called either. I managed to apply the new configuration object, and in the phone settings I see that the analysis of the submitted keys with the new configuration went well and multiple matches are found, however, `ExposureStateUpdatedAsync()` on the handler is never called.

No exceptions are thrown.

<img width="572" alt="image" src="https://user-images.githubusercontent.com/14860255/110816109-56027a00-828b-11eb-8b28-fe03080940b9.png">

@mattleibow 

@tmurakami Did you test it out as well? Do you have the same issue?
Thanks in advance!

Reply:
@pavshr I'm not sure because I can't test it, but I guess you need to assign `Infectiousness` to all the keys (-14 to 14).
https://github.com/folkehelseinstituttet/Fhi.Smittestopp.App/blob/044831fe92695866643d5c14517aaaf2a00002c3/NDB.Covid19/NDB.Covid19/ExposureNotifications/ExposureNotificationHandler.cs#L168-L173

```csharp
dsc.DaysSinceOnsetInfectiousness = new Dictionary<int, Infectiousness>()
{
    [-14] = Infectiousness.Standard, // Standard, High, or None
    [-13] = Infectiousness.Standard,
    [-12] = Infectiousness.Standard,
    *snip*
    [-1] = Infectiousness.Standard,
    [0] = Infectiousness.Standard,
    [1] = Infectiousness.Standard,
    *snip*
    [12] = Infectiousness.Standard,
    [13] = Infectiousness.Standard,
    [14] = Infectiousness.Standard,
};
```

See **Configure Criteria to Estimate Risk** for more details.
https://developer.apple.com/documentation/exposurenotification/building_an_app_to_notify_users_of_covid-19_exposure

Reply:
@mattleibow The same results from my debugging. It seems detectionSummary.DaySummaries.Length is always 0 or even null.

Reply:
When using the V2 TEK format in the zip file (with DSOS and ReportType), iOS' `ExposureStateUpdatedAsync` is called. So there is no issue here, just managed to verify it yesterday.

FYI @tmurakami

Reply:
@ns-kano @pavshr
I had the same problem. I found out we can set the parameter `infectiousnessWhenDaysSinceOnsetMissing` to `infectiousnessForDaysSinceOnsetOfSymptoms`.

https://github.com/keiji/chino/commit/f6032cc45dc67340e77ae9a21cf94c1723c51f0b#diff-1d4d67341f260be73d3e12f741daf36c4e17020b81c566ca3a56cfc10fb0ca59R383-R391

You can see the code in `Server.swift`, a [reference app provided by Apple](https://developer.apple.com/documentation/exposurenotification/building_an_app_to_notify_users_of_covid-19_exposure).

I didn't try to V2 TEK format. I am using [TEK Export Tool from exposure-notifications-server](https://github.com/google/exposure-notifications-server/tree/main/tools/export-generate).